### PR TITLE
#182201677 Feature:reset-password

### PR DESCRIPTION
**Pull request by Placide123**
##  What does this PR do?
- Reset a password
##  Description of Task to be completed?
- design an interface will be used by a user to reset a password
## How should this be manually tested?
- Clone the repo.
- checkout to a branch called `ft-reset-password-182201677`.
- Run npm install to install modules.
- Run `npm start` to run an app.
- Find `locahost:8080/reset` and put your email to reset
-  Check your email then click on the link then put your password.

## pivot Tracker link
https://www.pivotaltracker.com/story/show/182201677
## Questions
- N/A.

### CODE REVIEW

**Review by joshwambere — CHANGES_REQUESTED** [review]
Screenshot from 2022-08-16 15-59-55


and also the feature is not working as expected 
Screenshot from 2022-08-16 16-01-01
Screenshot from 2022-08-16 16-01-01

**Comment by joshwambere** [comment]
> Screenshot from 2022-08-16 15-59-55
> 
> and also the feature is not working as expected Screenshot from 2022-08-16 16-01-01 Screenshot from 2022-08-16 16-01-01

also use storybook for re-usable components like inputs and buttons

**Review by uwambajeddy — COMMENTED** [review]
Responsiveness
Screenshot (494)
Screenshot (492)

**Review by uwambajeddy — APPROVED** [review]

**Review by mwisemarierose — APPROVED** [review]

File: src/components/App.js [hunk 1]
```diff
@@ -3,3 +3,20 @@
...
+// const App = () => {
+//   return (
+//     <div className="container">
+//       {/* <h1>Welcome to Barefoot Nomad Frontend</h1> */}
+//       {/* <Counter /> */}
+//       {/* <Reset /> */}
+//       {/* <Respassword /> */}
+//     </div>
+//   );
+// };
```
Comment (didasmbalanya) [inline src/components/App.js]: Why is App commented out ?

File: src/components/Button/Button.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Inside src/components/Button/Button.js
```
Comment (didasmbalanya) [inline src/components/Button/Button.js]: what is this comment for. ?

File: src/components/Input/Input.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+// import "../../styles/stories/Button.stories.scss";
```
Comment (didasmbalanya) [inline src/components/Input/Input.js]: Remove commented code if not needed

File: src/styles/stories/button.stories.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+   
+//     @media screen and (max-width: $tablet) {
+
+//         input {
+//             width: 350px;
+//             height: 40px;
+           
+//         }
+//     }
+// }
```
Comment (didasmbalanya) [inline src/styles/stories/button.stories.scss]: remove unused commented code

File: src/views/Respassword.js [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+    setFormValues({ ...formValues, [name]: value });
+  };
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    setFormErrors(validate(formValues));
+    setIsSubmit(true);
+  };
+
+  useEffect(() => {
+    console.log(formErrors);
```
Comment (didasmbalanya) [inline src/views/Respassword.js]: remove console.log

**Review by didasmbalanya — CHANGES_REQUESTED** [review]
What is this image for ? src/assets/placide.jpeg

**Pull request merged**